talent/hiring_process.md

```diff
@@ -0,0 +1,77 @@
+# Hiring Process Explained
+
+This document explains the internal hiring process at source{d} and the responsibilities for each person related to it.
+
+### Job Description
+
+- Job Description is written by the lead engineers with the Talent's Manager support. 
+
+- For every meeting between the lead and the Talent Manager there will be a PR in the [minutes](https://github.com/src-d/minutes) repo and the job descriptions will be uploaded at the private repository https://github.com/src-d/company.
+
+- CTO and CEO should be added as reviewers (but should not be bottlenecks to continue).
```

Comment:
bottlenecks? that implies it's a capacity problem
what does this mean exactly?

Reply:
Means to delay and pause the process. Will rephrase it.